src/main/java/ca/warp7/frc2020/subsystems/ControlPanelSpinner.java

```diff
@@ -35,4 +113,24 @@ public void setSpeed(double speed) {
     public ColorSensorV3 getColorSensor() {
         return colorSensor;
     }
+
+
+    //===TEST FUNCTIONS===
+
+    private void testRgbToHue(){
```

Comment:
Can you please move this to a test file. I know that the build is broken right now and the tests won't actually run, but I'll fix it later. Just want to keep the main code simple.

Reply:
the testing framework should run now. If you want to test it, you can do `git pull origin master` on your `ctrl-panel` branch, then running the tests